components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -350,6 +352,81 @@ function selectProcessForChart(){
     });
 }
 
+//User Performance Over time
+
+function selectUserForPerformance(){
+    var x = document.getElementById("selectUser").value;
+    var url = httpUrl + "/" + CONTEXT + "/stats?update=true&option=userperformance&id=" + x ;
+
+    $.ajax({
+        type: 'GET',
+        contentType: "application/json",
+        url: url,
+        success: function (data) {
+            var array = eval('('+data+')');
+            google.load("visualization", "1.1", {packages:["bar"]});
+            google.setOnLoadCallback(drawChart(array));
+
+            function drawChart(data) {
+
+                var dataArr = [['Month', 'Completed Tasks', 'Started Tasks']];
+                for(var i = 0;i < data.length;i++){
```

Comment:
Better to run code formatting as a best practice. This will add proper spaces between operators and avoid having unformatted code.